app/views/users/registrations/start_registration.html.erb

```diff
@@ -1,18 +1,22 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Register as the induction coordinator for your school</h2>
-    <p>In order to register, we need your email address</p>
-
-    <%= form_for(:induction_coordinator_profile, url: users_check_email_path) do |f| %>
+    <h2 class="govuk-heading-l">Register your school</h2>
 
+    <%= form_for(:induction_coordinator_profile, url: users_check_details_path) do |f| %>
+      <%= f.govuk_text_field(
+            :school_urn,
+            value: @urn,
+            hint: { text: "6 digit number", class: "govuk-hint" },
+            label: { text: "Your school's Unique Reference Number (URN)", class: "govuk-label govuk-label--s" })
+      %>
+      <p>><%= govuk_link_to "Find my URN", "#", class: "govuk-link" %></p>
       <%= f.govuk_email_field(
             :email,
-            label: { text: "Email address" },
-            hint: { text: "This should be the email address provided by your school" },
-            autocomplete: "email")
+            value: @email,
+            label: { text: "Your school email address", class: "govuk-label govuk-label--s" })
       %>
 
-      <%= f.govuk_submit "Sign up" %>
+      <%= f.govuk_submit "Continue" %>
     <% end %>
 
   </div>
```

Comment:
This file and `users/registrations/new` look remarkably similar. Can we commonize them a little? The form for example.

Reply:
I can't see anything we can commonise and I think trying to do so for things which are likely to need frequent changes like these two would probably not be a good idea.

Reply:
Well, the form is similar.
We have 
```
    <%= form_for(:induction_coordinator_profile, url: users_check_details_path) do |f| %>
      <%= f.govuk_text_field(
            :school_urn,
            value: @school&.urn,
            hint: { text: "6 digit number", class: "govuk-hint" },
            label: { text: "Your school's Unique Reference Number (URN)", class: "govuk-label govuk-label--s" })
      %>
      <p class="govuk-body govuk-!-font-weight-bold">
        <%# TODO replace with actual link when available %>
        <%= govuk_link_to "Find my URN", "#", class: "govuk-link" %>
    </p>
      <%= f.govuk_email_field(
            :email,
            value: @user&.email,
            autocomplete: "email",
            label: { text: "Your school email address", class: "govuk-label govuk-label--s" })
      %>
      <%= f.govuk_submit "Continue" %>
    <% end %>
```
in the other one. The only differences I can spot are that in `registrations/new`, we are passing urn and email through other objects (user and school), while here they are their own variables.

Well, that and one has a TODO on a dead link, and the other doesn't. 

If you think they will very likely change and it is just a coincidence they look the same I'm happy to let them be coincidentally similar though :)